configs/processors/arxiv_unified_safe.yaml

```
@@ -0,0 +1,185 @@
+  save_frequency: 10   # Save every 10 papers for safety
+  staging_dir: /bulk-store/arxiv-staging
+  compress_staging: true
+  cleanup_staging: true
+
+# Logging configuration
+logging:
+  level: INFO
+  file: logs/unified_pipeline_safe.log
+
+# Monitoring configuration  
+monitoring:
```

YAMLlint (1.37.1)

[error] duplication of key "monitoring" in mapping

(key-duplicates)